### ISSUE TITLE
Ludwig predicting is giving the same value

### ISSUE THREAD

**Issue by aminaBm** [uber/ludwig#420]
Hello, so this is my YAML file
```
training:

    batch_size: 32
    epochs: 1000
    early_stop: 50
    learning_rate: 0.001
    optimizer:
        type: rmsprop
input_features:
    -
        name: content
        type: sequence
    -
        name: score
        type: category
output_features:
    -
        name: score
        type: numerical
        optimizer:
            type: mean_squared_error
        num_fc_layers: 2
        fc_size: 64
```
Why it's always giving me the same value upper than 1?
for any prediction line in input, it gives me this value = 1.16239...?
how to do to have a value between 0 and 1 and make it more accurate?

**Comment by w4nderlust** [comment]
Hi @aminaBm , that really depends on your data, setting hyperparameters for your specific use case it's what you have to figure out :)
Conversely, if you believe there's a bug that is making the model not working for your usecase, than that's an issue we can try to solve.

By the way, there's an error in you model definition, the `optinizer` in your output feature should be `loss` instead, but it should work anyway.

**Comment by w4nderlust** [comment]
I'm interpreting the thumbs up as "I solved the issue". If that's not the case, please comment again and I will reopen. If the issue persists, please provide a full reproducible example, either yaml+command+data or python+data. If the data is big or you can't share it a sample is fine too, or even a synthetic dataset obtained through `data/dataset_synthesyzer.py` works, as long as I can reproduce the problem i can try to debug it.

**Comment by aminaBm** [comment]
I a beginner just I want to get a score value that varies between 0 and 1 based on the content feature I changed the code as you see next and I still give me values upper to 1 for some sentences I didn't get the reason 
`input_features:
    -
        name: content
        type: sequence
output_features:
    -
        name: score
        type: numerical
        num_fc_layers: 2
        fc_size: 64`

**Comment by w4nderlust** [comment]
You can either use the `clip` functionality to clip the scores.
Or are those scores actually probabilities? In that case maybe you may want to use a binary output feature.
